packages/freeimage/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE=https://freeimage.sourceforge.io
+TERMUX_PKG_DESCRIPTION="FreeImage is a library project for developers who would like to support popular graphics image formats."
+TERMUX_PKG_LICENSE="GPL-3.0, GPL-2.0, other"
```

Comment:
`other` is also not a valid license, you'll have to use `custom` for the freeimage license. 
The license files should then also be included in the package, since it is not part of the standard ones, which can be done by adding `TERMUX_PKG_LICENSE_FILE="license-fi.txt, license-gplv2.txt, license-gplv3.txt"`.